pkg/telemetry/client_test.go

```diff
@@ -0,0 +1,138 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+package telemetry
+
+import (
+	"errors"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	monkit "gopkg.in/spacemonkeygo/monkit.v2"
+)
+
+func TestNewClient_IntervalIsZero(t *testing.T) {
```

Comment:
@crawter I'd recommend reading about Table Driven Tests: https://dave.cheney.net/2013/06/09/writing-table-driven-tests-in-go

We use this pattern widely in the code base.

Reply:
@kaloyan-raev Thx! I'll use this pattern in future!